mail/protocols/imap/src/main/java/com/fsck/k9/mail/store/imap/ImapFolder.java

```diff
@@ -690,6 +704,7 @@ public void fetch(List<ImapMessage> messages, FetchProfile fetchProfile,
             try {
                 String commaSeparatedUids = ImapUtility.join(",", uidWindow);
                 String command = String.format("UID FETCH %s (%s)", commaSeparatedUids, spaceSeparatedFetchFields);
+                Timber.d("Sending command "+command);
```

Comment:
I assume this is left over from debugging. Please remove.

Reply:
Oh, yes.